Camera.hpp

```diff
@@ -6,6 +6,13 @@
 
 namespace spic {
 
+    struct RenderTarget {
```

Comment:
Is dit niet gewoon een spic::Point?

Reply:
Eh nu je t zegt ja, wil je dat ik dit verander in een `using RenderTarget = spic::Point;` om de naam wel duidelijk te houden?

Reply:
T enige verschil is dat dit ints zijn en die ander doubles